Unix/installbuilder/datafiles/Linux.data

```diff
@@ -328,6 +330,22 @@ if ${{PERFORMING_UPGRADE_NOT}}; then
 fi
 
 %Postinstall_1500
+
+# Special handling for Red Hat 5 don't have open selinux permission
+# return 0, means is redhat 5; 1 means others.
+is_redhat5(){
```

Comment:
Do we need this we no longer support RedHat5

Reply:
our universal build for rpm and deb is redhat 5.

Reply:
I don't know if we can drop it or not.
@sarojcare how do you think?

Reply:
I see 2016 UR 7 support RedHat 5.

Reply:
we should think of upgrading our universal build for rpm and deb from RedHat 5

Reply:
due to some history reason, we still use RedHat 5 for build server.
an old machine list:

Supported Platforms | glibc | Machine
-- | -- | --
Centos5x64(Not support) | glibc-2.5-123.el5_11.3 | osd64-ct5-01
centos6x64 | glibc-2.12-1.209.el6_9.2.x86_64 | osd64-ct61-01
oracle6x64 | glibc-2.12-1.7.el6.x86_64 | It seems we don’t have pbuild for it.
Rhel5x64 | glibc-2.5-12 | osd64-rh5-01
Rhel6x64 | glibc-2.12-1.7.el6.x86_64 | osd64-rh6-01